deployment/stacks/ui-auth.ts

```diff
@@ -18,26 +18,27 @@ interface CreateUiAuthComponentsProps {
   scope: Construct;
   stage: string;
   isDev: boolean;
-  oktaMetadataUrl: string;
   applicationEndpointUrl: string;
   restApiId: string;
-  bootstrapUsersPasswordArn: string;
-  iamPermissionsBoundary: IManagedPolicy;
-  iamPath: string;
   customResourceRole: iam.Role;
+  iamPath: string;
+  iamPermissionsBoundary: IManagedPolicy;
```

Comment:
We're hardcoding this and iamPath now, right?  If so, why do we still have as props?

Reply:
oh, gotcha.  either way.  Maybe instead of passing it in, we could just const iamPermissionsBoundary at the beginning and reuse many times to only have the ugly string once.